File: Examples/TextContinuationExample.cs
using ceTe.DynamicPDF;
using ceTe.DynamicPDF.PageElements;
using DynamicPDFCoreSuite.Utility;

namespace DynamicPDFCoreSuite.Examples
{
    public class TextContinuationExample
    {
        public static void Run()
        {
            Document document = new Document();

            TextArea textArea = new(TextGenerator.GenerateLargeTextDoc(), 0, 0, 512, 692,
                Font.Helvetica, 14);

            do
            {
                Page page = new Page();
                page.Elements.Add(textArea);
                document.Pages.Add(page);
                textArea = textArea.GetOverflowTextArea();

            } while (textArea != null);
                      
            document.Draw(Util.GetPath("Output/text-area-overflow-example.pdf"));
        }
    }
}

File: dynamicpdf-vbnet-examples/Examples/TextContinuationExample.vb
Imports ceTe.DynamicPDF
Imports ceTe.DynamicPDF.PageElements
Imports dynamicpdf_vbnet_examples.DynamicPDFCoreSuite.Utility
Imports DynamicPDFCoreSuite.Utility

Namespace DynamicPDFCoreSuite.Examples
    Public Class TextContinuationExample
        Public Shared Sub Run()
            Dim document As New Document()

            Dim textArea As New TextArea(TextGenerator.GenerateLargeTextDoc(), 0, 0, 512, 692, Font.Helvetica, 14)

            Do
                Dim page As New Page()
                page.Elements.Add(textArea)
                document.Pages.Add(page)
                textArea = textArea.GetOverflowTextArea()
            Loop While textArea IsNot Nothing

            document.Draw(Util.GetPath("Output/text-area-overflow-example.pdf"))
        End Sub
    End Class
End Namespace

